src/documentation/index.html

```diff
@@ -31,7 +31,7 @@
 
     <!--#include virtual="/assets/includes/head-assets.html"-->
 
-    <link href="/assets/v3/css/qg-documentation.css" rel="stylesheet" type="text/css" media="all">
+    <link href="/assets/_project/css/qg-documentation.css" rel="stylesheet" type="text/css" media="all">
```

Comment:
Why we have _project as the path?  I think this needs to be v3 ? How this will work after we upload stuff on cdn under the v3?